addons/hr_expense/hr_expense_demo.xml

```diff
@@ -35,61 +35,60 @@
             <field name="name">Hotel Accommodation</field>
             <field name="default_code">HA0</field>
             <field name="categ_id" ref="hr_expense.cat_expense"/>
-            <field name="hr_expense_ok" eval="True" />
+            <field name="can_be_expensed" eval="True"/>
             <field name="image" type="base64" file="hr_expense/static/img/hotel_rent-image.jpg"/>
         </record>
-        <record id="expenses0" model="hr.expense.expense">
-            <field name="name">Expenses</field>
+
+        <record id="expenses0" model="hr_expense.sheet">
+            <field name="name">Expenses for Administrator</field>
+            <field name="employee_id" ref="hr.employee"/>
             <field name="date" eval="time.strftime('%Y-08-25')"/>
+            <field name="employee_payable_account_id" ref="account.a_pay"/>
         </record>
-        <record id="expenses_line0" model="hr.expense.line">
+
+        <record id="expenses_line0" model="hr_expense.expense">
             <field name="name">Travel Expenses</field>
-            <field name="date_value" eval="time.strftime('%Y-08-10')"/>
             <field name="analytic_account" ref="analytic.analytic_consultancy"/>
             <field name="expense_id" ref="expenses0"/>
             <field eval="200.0" name="unit_amount"/>
             <field name="uom_id" ref="product.product_uom_unit"/>
             <field eval="1.0" name="unit_quantity"/>
+            <field name="state">confirm</field>
```

Comment:
Why expense directly in 'confirm' state ???
You can confirm expense by: 
<workflow action="expense_confirm" model="hr_expense.expense" ref="expenses_line0"/>